### DISCUSSION TITLE
Stopped working after 2022.5.5

### DISCUSSION THREAD

**Discussion by gulo101** [cytech/Home-Assistant-wundergroundpws#101] [General]
Hi

I just upgraded to 2022.5.5 yesterday, I think I skipped couple versions. Since the upgrade the wundergroups stopped working for me.

I have not changed any settings. 

![image](https://user-images.githubusercontent.com/11531549/169896016-6923954a-6018-482b-a58c-89b24573e160.png)

This is the error I get:

Error while setting up wundergroundpws platform for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 249, in _async_setup_platform
    await asyncio.shield(task)
  File "/config/custom_components/wundergroundpws/sensor.py", line 374, in async_setup_platform
    await rest.async_update()
  File "/config/custom_components/wundergroundpws/sensor.py", line 538, in async_update
    with async_timeout.timeout(10, loop=self._hass.loop):
TypeError: timeout() got an unexpected keyword argument 'loop'

My config:

  - platform: wundergroundpws
    api_key: ########
    pws_id: ###
    numeric_precision: none
    monitored_conditions:
        - uv
        - humidity
        - dewpt
        - pressure
        - temp
        - precip_1d
        - precip_chance_1d
        - temp_high_1d
        - temp_low_1d
        - temp_high_2d
        - temp_low_2d
        - temp_high_3d
        - temp_low_3d
        - temp_high_4d
        - temp_low_4d
        - temp_high_5d
        - temp_low_5d
        - wind_1d
        - weather_1d
        - weather_1n
        - weather_2d
        - weather_2n
        - weather_3d
        - weather_3n
        - weather_4d
        - weather_4n
        - weather_5d
        - weather_5n

**Comment by cytech** [comment]
issue #97 
Please make sure you are running the latest release of the Home-Assistant-wundergroundpws custom addon

0.8.1

**Reply by gulo101** [reply]
That fixed it, I was on 0.8.0. Thanks!

**Comment by cytech** [comment]
closed as outdated

**Comment by cytech** [comment]
closed as outdated
